tests/cupyx_tests/scipy_tests/signal_tests/test_resample.py

```diff
@@ -190,6 +190,22 @@ def test_resample_methods(self, method, ext, padtype, xp, scp):
 
         return results
 
+    @testing.numpy_cupy_allclose(scipy_name='scp')
+    def test_resample2d(self, xp, scp):
+        results = []
+        for nx in range(10000, 100000, 10000):
+            for down in [2, 3, 8]:
+                for ny in range(1, 50, down):
+                    for axis in [0, 1]:
```

Comment:
@IdanFried are you still happy to update this?  I am happy to just put this in, but this test needs to be thinned out by a factor of ~100 (or more).

Running it locally, it takes 85 seconds and I don't see that is needed, CI is expensive, so let's try to get it to <1s and if a bit more extensive testing is really relevant than that may be OK but with a `@testing.slow` marker.